### ISSUE TITLE
Feature request: add a mode for creating color by dataset 'label'.

### ISSUE THREAD

**Issue by landure** [kurkle/chartjs-plugin-autocolors#24]
Thank you for your work on autocolors. It would be nice if autocolors was able to use the same color when a dataset label /  legend is encountered multiple times.
One use-case for this is for combined graphs (bars + lines) on the same chart. 

An example of datasets that could use this feature is bellow. Autocolors creates 3 colors for this example, although the chart would be better suited with 2 colors based on the dataset labels.

```js
datasets = [
{
  'label': 'seller name',
  'type': 'bar',
  'data': [ 'material': 20, 'other-material':  3 ]
  'xAxisID': 'by-name',
  'yAxisID': 'total',
  'stack': 'by-name',
},
{
  'label': 'other seller name',
  'type': 'bar',
  'data': [ 'material': 20, 'other-material':  3 ]
  'xAxisID': 'by-name',
  'yAxisID': 'total',
  'stack': 'by-name',
},
{
  'label': 'seller name',
  'type': 'line',
  'data': [ '2022-01': 15, '2022-02': 5 ]
  'xAxisID': 'by-month',
  'yAxisID': 'orders',
  'stack': 'by-month',
},
]
```

**Comment by kurkle** [comment]
Should be quite easy to implement, just a new 'label' mode, using a map for the colors.

**Comment by kurkle** [comment]
Implemented in 35d94e4c4c704b5ca7aeaea009e94c6b3506e4b1, shortly available on version 0.2.0

**Comment by landure** [comment]
Thank you for your quick action and for your work.
